test/js-corpus.go

```diff
@@ -0,0 +1,154 @@
+package main
```

Comment:
Nit: I'd move this script into `tools`.

Nit: I'd call this script something like `download-js-corpus.go`.